docs/source/tutorials/extending_mantid_with_python/python_algorithms_and_workspaces/06_exercise_3.rst

```diff
@@ -0,0 +1,49 @@
+.. _06_exercise_3:
+
+==========
+Exercise 3
+==========
+
+The aim of this exercise is to write a small algorithm that wraps a small
+script that focuses some powder diffraction data.
+
+Write an algorithm called ``PowderDiffractionReduce``. The algorithm should
+have 3 properties:
+
+* ``Filename``: A FileProperty for a TOF data file to load (ignore extensions)
+* ``CalFilename``: A FileProperty for a cal file (ignore extensions)
+* ``OutputWorkspace``: An output WorkspaceProperty to hold the final result.
+
+The steps the algorithm should perform are:
+
+#. Use the :ref:`Load <algorithms.Load.v1>` algorithm to load the TOF
+   data
+#. Apply the calibration file using
+   :ref:`ApplyDiffCal <algorithms.ApplyDiffCal.v1>`
+#. Run :ref:`ConvertUnits <algorithms.ConvertUnits.v1>` on the TOF data
+   to convert to ``dSpacing``
```

Comment:
@DannyHindson here's the related exercise instructions if you can check these make sense. The old exercise is here:  https://www.mantidproject.org/Extending_Mantid_With_Python__Exercise_3.html

Reply:
Where it says "using the same cal file as the previous step". It's actually the previous step but one now so strictly speaking it should say "earlier step" perhaps. But no big deal if you'd like to get this merged. Looks fine apart from that

Reply:
Good catch, I will quickly update this now